packages/commerce-sdk-react/src/hooks/useQuery.ts

```diff
@@ -55,17 +69,16 @@ export const useQuery = <Client extends ApiClient, Options extends ApiOptions, D
     // for this case would add significantly more complexity.
     const wrappedMethod = async () => await authenticatedMethod(apiOptions as Options)
 
-    return useReactQuery({
+    const finalEnabled =
```

Comment:
NIT: Interesting variable name choice. The current context of this method would imply that if you are creating any type of variable to indicate if it's overall "enabled-ness" of the hook being used.. so `enabled` or `isEnabled` is probably enough. `finalEnabled` just doesn't read to well in the realm of the english language .

Reply:
good call, fixed